velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -460,7 +464,7 @@ void ExpressionFuzzer::appendConjunctSignatures() {
 }
 
 RowVectorPtr ExpressionFuzzer::generateRowVector() {
-  return vectorFuzzer_.fuzzRow(
+  return vectorFuzzer_.fuzzInputRow(
```

Comment:
I know we have this assumptions that top-level row vectors won't have nulls, but is this breaking something? It also looks a bit orthogonal to this PR.

Reply:
Currently the way the velox works, we only have lazyVectors as children in row vectors is when its an input vector (no null children). Ideally expressions are also usually called on input vectors so I thought this would be ok.
I have considered a few arguments for an against this change, let me know what you think:

For (using fuzzInputRow):
- More closely match how we usually run expressions
- columns (to wrap in lazy) can be randomly selected without worrying about nullability. Otherwise, having null columns would result in not hitting the probability of selecting lazy columns.

Against (using fuzzRow):
- We gain coverage on running the fuzzed expression on a vector that can represent a column of row type with possible null children. Not sure how much coverage we will get here vs using fuzzInputRow() which can still have children that are rows.

I dont have a strong preference either way as the arguments are more about justifying randomness which is a hard thing to prove or compare in this instance.

Reply:
No, it would still work, just saying that if we use "RowVectors with null top-level" we wont hit the targeted proportion of columns that we want to wrap in lazy ().